src/lib/activity-store.js

```diff
@@ -3,10 +3,11 @@ import Elasticsearch from '@elastic/elasticsearch';
 
 class ActivityStore {
 
-  constructor(esIndex){
+  constructor(esIndex, esIndexMeta){
     this.user = process.env.ELASTICSEARCH_USERNAME || "";
     this.pass = process.env.ELASTICSEARCH_PASSWORD || "";
     this.esIndex = esIndex;
+    this.esIndexMeta = esIndexMeta;
```

Comment:
Could we call it esHavesterStateIndex or something like that, just because it might be confused with OA metadata

Reply:
ok